#!/usr/bin/env python
PACKAGE = "project"


from dynamic_reconfigure.parameter_generator_catkin import *
gen=ParameterGenerator()

gen.add("select_P",int_t,0,"select between the two odometries",0,0,1)
gen.add("x_P",int_t,0,"Xcoordinate",0)
gen.add("y_P",int_t,0,"Ycoordinate",0)
gen.add("theta_P",int_t,0,"thetaCoor",0,0,360)
gen.add("reset_odom",bool_t,0 ,"reset_command",False)

exit(gen.generate(PACKAGE,"dyn_reconfigure","parameters"))
